test/lib/foreman/renderer_test.rb

```diff
@@ -49,6 +41,38 @@ def setup_safemode_renderer
       assert_equal 'bar', tmpl
     end
 
+    test "foreman_server_fqdn helper method" do
```

Comment:
That's can't be created actually, the setup block creates a closure around renderer_name and it's not set appopriately.